kuksa-client/README.md

```diff
@@ -82,12 +81,59 @@ Test Client> authorize /some/path/kuksa_certificates/jwt/super-admin.json.token
 
 If connecting to Databroker the command `printTokenDir` is not much help as it shows the default token directories
 for KUKSA.val Server example tokens. If the KUKSA.val Databroker use default example tokens then one of the
-tokens in [../jwt](..//jwt) can be used, like in the example below:
+tokens in [../jwt](../jwt) can be used, like in the example below:
 
 ```console
-Test Client> authorize /some/path//jwt/provide-all.token
+Test Client> authorize /some/path/jwt/provide-all.token
 ```
 
+## Using TLS
```

Comment:
structure wise comment: ```Using TLS``` and then ```Connecting to KUKSA.val Server``` as sub headline without having a Connecting to KUKSA.val Databroker is not the best way. Only my opinion and nothing 'big'

Reply:
Good point, will fix it as soon as I see that all builds pass